README.md

```diff
@@ -8,6 +8,12 @@ This program shows where time and bytes are spent when building a docker context
 It is based directly on the same logic that Docker itself uses to build the
 context.
 
+## Getting started (docker):
+
+```sh
+docker run -v $PWD:/data -w /data sanfilippopablo:/docker-show-context
```

Comment:
Sorry, I don't want to point users at a binary that I'm not personally supporting (actually, I'd rather not point users at binaries in general).

I'd accept a `docker run -v $PWD/data $(docker build https://github.com/pwaller/docker-show-context)` though, which is almost the same length but grabs the sources from git. Unfortunately won't work until there is a Dockerfile in master...

(Also suggest moving `WORKDIR /data` into the Dockerfile rather than supplying `-w`).

Reply:
Makes sense. Let me try that